.yamato/wrench/pvp-exemptions.json

```diff
@@ -0,0 +1,29 @@
+{
+  "exempts": {
+    "PVP-20-1": {
+      "errors": [
+        "Missing Doc on System.String Unity.Cinemachine.Editor.CinemachineTool::GetIconPath()",
```

Comment:
Why are these exemptions here?

Reply:
These are the existing  PVP exemptions taken from here : https://github.com/Unity-Technologies/com.unity.cinemachine/blob/f38e9bddb4361158f27a43ca01d37573572f0656/pvp_api_doc_exemptions.json and moved to where the wrench jobs will find them. We could delete the original copy so there is no duplication.

Reply:
These doc exceptions are all stale.  I will remove them altogether